erpnext_documentation/www/docs/user/manual/en/selling/blanket-order.md

```diff
@@ -23,27 +23,29 @@ Before creating and using a Blanket Order, it is advised that you create the fol
 1. Enter Item Code, Quantity and Rate in the Item table. You can also mention the Terms and Conditions for each item.
 1. Submit.
 
-<img class="screenshot" alt="Opening Account" src="{{docs_base_url}}/assets/img/selling/blanket_order_sales.gif">
+<img class="screenshot" alt="Blanket Order Selling" src="{{docs_base_url}}/assets/img/selling/blanket-order-sales.gif">
```

Comment:
there's something wrong with your GIF. 5.24MB and still has terrible quality. Use LICEcap!!!